### PR TITLE
Created empty CI/CD Integration doc page.

### PR DESCRIPTION
**Pull request by jfeingold35**

### CODE REVIEW

File: docs/_articles/en/v3.x/ci-cd-integration.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: CI/CD Integration
+lang: en
+redirect_from: /en/ci-cd-integration
+---
+<<<<<<< Updated upstream
+=======
```
Comment (jfeingold35) [inline docs/_articles/en/v3.x/ci-cd-integration.md:7]: @teresa-allen-sfdc , you're going to want to remove lines 6 and 7 here.

File: docs/_articles/en/v3.x/ci-cd-integration.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Accelerate Your CI/CD Integration
+
+Our community of users continues to develop templates and tools that help you speed up Code Analyzer integration into your CI/CD process.
+
+Try these community tools:
+
+* [SFDX Scan Pull Request](https://github.com/marketplace/actions/sfdx-scan-pull-request) by Mitchell Spano
+* [Pull Request SFDX Code Review](https://github.com/marketplace/actions/pull-request-sfdx-code-review) by Aleš Remta
+>>>>>>> Stashed changes
```
Comment (jfeingold35) [inline docs/_articles/en/v3.x/ci-cd-integration.md:32]: @teresa-allen-sfdc , you're also going to want to remove line 32 here.

**Review by teresa-allen-sfdc — COMMENTED** [review]
@jfeingold35 Edited per your comments
Ready for approval @rmohan20

File: docs/_articles/en/v3.x/ci-cd-integration.md (hunk 1 above)
Comment (teresa-allen-sfdc) [reply docs/_articles/en/v3.x/ci-cd-integration.md]: @jfeingold35 Done

File: docs/_articles/en/v3.x/ci-cd-integration.md (hunk 2 above)
Comment (teresa-allen-sfdc) [reply docs/_articles/en/v3.x/ci-cd-integration.md]: @jfeingold35 Done

**Review by rmohan20 — APPROVED** [review]

**Pull request merged**